lib/gitlab/client/projects.rb

```diff
@@ -326,11 +326,15 @@ def delete_deploy_key(project, id)
     #
     # @example
     #   Gitlab.create_fork(42)
+    #   Gitlab.create_fork(42, :sudo => 'another_username')
     #
     # @param  [Integer] project The ID of a project.
+    # @param  [Hash] options A customizable set of options.
+    # @option options [String] :sudo The username the project will be forked for
     # @return [Gitlab::ObjectifiedHash] Information about the forked project.
-    def create_fork(id)
-        post("/projects/fork/#{id}")
+    def create_fork(id, options = {})
+      body = {}.merge(options)
```

Comment:
Lets skip the merge here.  Just pass `options` directly as the `body` and if we need to do a merge later we can do it.